docs/source/documentation.html.haml

```diff
@@ -0,0 +1,130 @@
+---
+title: Algolia Places - Documentation
+---
+%div.navigation
+  %div.fl-left
+  %a{href: "/index.html"}
+    %span.brand
+      <span class="icon icon-algolia-light">Algolia</span> Community
+    %span.pipe
+    %span.no-mobile Places
+
+
+  %div.fl-right
+    %div.searchbar
+      %input{class: "form-control aa-input", placeholder: "Search in the documentation", type: "text", id: "doc-search"}
+
+    %ul.menu
+      %li.active
+        %a{href: "/documentation.html"} documentation
+      %li
+        %a{href: "example.html"} examples
+      %li
+        %a{href: "#"} about
+      %li
+        %a{href: "#"}
+          %span.icon.icon-github
+            github
+
+
+%section.hero-section.shrink
+  .container
+    %div.hero-container
+
+      %div.fl-left
+        %span{class: "icon icon-algolia-small no-mobile", id: "icon-algolia"}
+        %span{class: "display-on-small small-illustration"}
+          %img{src: "images/svg/places-illustration.svg"}
+        %h1 Documentation
+        %h3 The easiest way to find an address lorem dolor sit amet en consequeter
+
+
+%section.documentation-section
+  .container
+    %nav.sidebar#sidebar
+      %ul
+        %li
+          %a{href: "#introduction"} Introduction
+
+        %li
+          %a{href: "#setup"} Setup
+
+        %li
+          %a{href: "#options"} Options
+
+    %div.documentation-container
+      %article{id: "introduction"}
+        %h2 Introduction
+
+        %p
+          We’re scratching our own itch here. As developers, we spend a lot of time
+          reading documentation, and it isn’t always easy to find the information we
+          need.
+
+        %p
+          Not blaming anyone here. Building a good search for a documentation is a
+          complex challenge. We happen to have a lot of experience doing that, and we
+          want to share it with the world. For free.
+
+        %p
+          Just submit the form on the website and we’ll get back to you with what you
+          need to integrate your new search into your website.
+        %ul
+          %li We’ll crawl your documentation pages, 
```

Comment:
No, at the end of this line (73), you have some blank characters

`%li We’ll crawl your documentation pages, <<<<here`

see my system shows it because of the fonts I am using:

![2016-03-30-142313_475x44_scrot](https://cloud.githubusercontent.com/assets/123822/14141987/10d8b4ba-f683-11e5-8af0-fa9b57ad2b0d.png)